### DISCUSSION TITLE
REST API playtime

### DISCUSSION THREAD

**Discussion by WolverStones** [plan-player-analytics/Plan#3019] [APIs and Development]
Hello, I have a question. Is it possible to obtain from /v1/network/overview the numbers.playtime field only for the "day" parameter? By that, I mean getting the time in days indicating how many days players haven't played.

**Comment by AuroraLS3** [comment]
The playtime is formatted in the backend, so unfortunately not.

In the future there are plans of moving the time and date formatting code to the frontend, at which point /v2/ version of the endpoint will become available with the raw values rather than formatted.

----

In the meanwhile you can parse the formatted time to days

Default time format is
`{} year(s), {} month(s), {}d {}h {}m {}s` where s in `(s)` is shown is if n is >1, and if some component is 0 it is not shown at all (Except when playtime is 0s).

year = 365d
month = 30d

**Reply by WolverStones** [reply]
Thanks. I'll look into it.
